core/integration/storagetest/batch.go

```diff
@@ -0,0 +1,128 @@
+package storagetest
+
+import (
+	"context"
+	"testing"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/google/go-cmp/cmp"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	spb "github.com/google/keytransparency/core/sequencer/sequencer_go_proto"
+)
+
+// Batcher writes batch definitions to storage.
+type Batcher interface {
```

Comment:
Why not just reference this same interface defined in `core`? I don't see a clear benefit of copy-pasting it to yet another place, esp. to tests. The idea should be presumably testing **the** interface, not some made-up interface that happens to be a superset. Also, keeping them in sync is an extra effort.

Reply:
Although not having to import that package is kinda nice...

Reply:
I'll make it a type alias